assets/src/components/d3/createResourceAccessChart.js

```diff
@@ -146,12 +149,35 @@ function createResourceAccessChart ({ data, width, height, domElement }) {
   const miniYScale = d3.scaleBand().range([0, miniHeight])
 
   const textScale = d3.scaleLinear().range([12, 6]).domain([15, 50]).clamp(true)
+  const unReadResources = resourceData.filter(resource => resource.self_access_count === 0)
+  // d3 graph narrative text
+  const narrativeTextResources = {}
+  narrativeTextResources.weekRange = `Resources accessed from week ${weekRange[0]} to ${weekRange[1]}.  `
+  narrativeTextResources.resourceType = `Selected resource type is ${resourceType}.  `
```

Comment:
I'm looking for a course with multiple types of data what happens when you have more than one type selected?